CHANGELOG.md

```diff
@@ -21,8 +21,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Move `eslint` to devDependencies ([#489][] @gmfvpereira)
+- Display wrong numbers in Person Contact Kit ([#499][] @KFad11)
```

Comment:
Almost there :) Now you're under the `## [3.0.1] - 2019-12-06` release section, you want to put it under the `## [Unreleased]` one.

I know it's not very obvious.

There's an `### Added` category there, you'll need to add a `### Fixed` one, like:

```
## [Unreleased]
### Added
- Introduce CHANGELOG file ([#498][] @rafbgarcia)
- Include Card Separator example in Card kit docs ([#439][] @rruiz85)
[#498]: https://github.com/powerhome/playbook/pull/498
[#439]: https://github.com/powerhome/playbook/pull/439

+ ### Fixed
+
+ - Display wrong numbers in Person Contact Kit ([#499][] @KFad11)
+
+ [#499]: https://github.com/powerhome/playbook/pull/499
```

Notice the `+` (plus) are your changes.

Reply:
So... close! lol. I find it way harder to read in my text editor. I should have given it a closer look in Github, then found my spot in the text editor.

Reply:
I think I was just looking for the first `### Fixed` and adding my fix to that list. I added the new `### Fixed`, right under the current unreleased `### Added` group.